.github/workflows/release.yml

```diff
@@ -39,12 +39,11 @@ jobs:
         run: |
           git config user.email "swagger-bot@smartbear.com"
           git config user.name "swagger-bot"
-          npx lerna version --no-private --yes
+          npx lerna version prerelease --preid alpha --no-private --yes
         env:
           GH_TOKEN: ${{ secrets.SWAGGER_BOT_GITHUB_TOKEN }}
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
       - name: Publish monorepo npm packages
         run: npx lerna publish from-package --no-private --yes
-        working-directory: ./packages/apidom-ast
```

Comment:
Removed this as this was just a forgotten fragment from previous testing of the CD pipeline.